### PR TITLE
Update Examples page: LTFU Workflow

### PR DESCRIPTION
**Pull request by Phil-Mwago**

### CODE REVIEW

**Comment by mrjones-plip** [comment]
Hey @Phil-Mwago !  We use "assign" field for who is doing the work on the ticket.  We use "Reviewers" for who is reviewing the ticket.

you have assigned me the ticket - would you like me to work on it with you?

**Comment by Phil-Mwago** [comment]
> 

@mrjones-plip yes, we can work together on this. Kindly review.

File: content/en/apps/examples/interoperability.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Loss to Follow Up Workflow (LTFU)
+
+This workflow describes a use case where a health facility/requesting system generates a list of patients who have missed appointments for follow-up through on the CHT. A Community Health Worker (CHW) physically visits the patients who missed the appointments and encourages them to visit the health facility. The follow-up process can also be conducted over phone or text messaging.
```
Comment (esthermmoturi) [inline content/en/apps/examples/interoperability.md:3]:
```suggestion
This workflow describes a use case where a health facility or a requesting system generates a list of patients who have missed follow-up appointments that were made through the CHT. A CHW would then follow up with the listed patients through SMS, a physical visit or a phone call.
```

File: content/en/apps/examples/interoperability.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Loss to Follow Up Workflow (LTFU)
+
+This workflow describes a use case where a health facility/requesting system generates a list of patients who have missed appointments for follow-up through on the CHT. A Community Health Worker (CHW) physically visits the patients who missed the appointments and encourages them to visit the health facility. The follow-up process can also be conducted over phone or text messaging.
+
+## Problem Being Addressed
+
+Data exchange between the CHT and other systems has primarily been at peer-to-peer level. This means that the integration is built to meet the specific need in the unique scenarios. This presents a problem during maintainance and scalability as there are no defined standards that have been used. Interoperability allows technical teams to scale in an efficient and repeatable manner due to the already predefined standards.
```
Comment (esthermmoturi) [inline content/en/apps/examples/interoperability.md:7]:
```suggestion
Data exchange between the CHT and other systems has primarily been at peer-to-peer level. In most cases, integration is built to meet the specific needs on case-by-case basis. This limits maintenance and scalability as there are no defined standards that are used. Interoperability allows technical teams to scale in an efficient and repeatable manner due to the predefined standards. 

```

File: content/en/apps/examples/interoperability.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Solution Overview
+This is a model for interoperability that can be used for Loss to Follow up flows between the CHT and a health facility or a requesting system. Community health workers routinely follow up on patients physically at their residences or place of work. In instances where the patient needs to visit a health facility for routine checkups or specialized care, the CHW has no visibility of the process outside the CHT. The interoperability layer built allows other systems including health facilities to exchange data in a standardized format.
+
+## Intended Users 
+The intended users of the interoperable systems are Community Health Workers (CHWs) on the CHT side and Healthcare Givers on the requesting system side. System administrators can access the mediator on the administration console.
```
Comment (esthermmoturi) [inline content/en/apps/examples/interoperability.md:13]:
```suggestion
The intended users of the interoperable systems are CHWs on the CHT side and Healthcare Givers on the requesting system side. System administrators can access the mediator on the administration console. 

```

File: content/en/apps/examples/interoperability.md (hunk 2 above)
Comment (esthermmoturi) [reply content/en/apps/examples/interoperability.md:7]:
```suggestion
Data exchange between the CHT and other systems has primarily been at peer-to-peer level. This means that the integration is built to meet the specific need in the unique scenarios. This presents a problem during maintainance and scalability as there are no defined standards that have been used. [Interoperability](https://docs.communityhealthtoolkit.org/apps/concepts/interoperability/) allows technical teams to scale in an efficient and repeatable manner due to the already predefined standards. 

```

**Review by esthermmoturi — CHANGES_REQUESTED** [review]
@mrjones-plip and @Phil-Mwago do you think we should get rid of the problem to be solved and solution overview subheadings?

**Comment by mrjones-plip** [comment]
Yeah - I like the feature parity and think we should keep "problem to be solved" and "solution overview"!

**Pull request merged**